src/lib.rs

```diff
@@ -49,12 +49,12 @@ pub use sqlx_sqlite::{self as sqlite, Sqlite, SqliteConnection, SqliteExecutor,
 #[cfg_attr(docsrs, doc(cfg(feature = "any")))]
 pub use crate::any::{reexports::*, Any, AnyExecutor};
 
-#[cfg(feature = "macros")]
+#[cfg(any(feature = "derive", feature = "macro"))]
```

Comment:
Why not just `#[cfg(feature = "derive")]`, since the macro feature activates that one?

Reply:
I could change to just `derive`, yes. Felt like using both would better transmit the idea, though.